sdk/eventhub/azure-eventhub-checkpointstoretable/azure/eventhub/_version.py

```diff
@@ -0,0 +1,2 @@
+# matches SEMVER
```

Comment:
move this file into the folder `azure-eventhub-checkpointstoretable/azure/eventhub/checkpointstoretable`

Reply:
actually this file should be removed since there is already a _version file in extensions/...